Atomic/pull.py

```diff
@@ -38,19 +38,23 @@ def pull_docker_image(self):
         #     pull_uri = 'atomic:'
         # else:
         #     pull_uri = 'docker://'
-        fq_name = self.get_fq_image_name(self.args.image)
-        registry, _, _, tag, _ = Decompose(fq_name).all
-        image = "docker-daemon:{}".format(self.args.image)
-        if not self.args.image.endswith(tag):
-            image += ":{}".format(tag)
-        insecure = True if is_insecure_registry(self.d.info()['RegistryConfig'], strip_port(registry)) else False
-        trust = Trust()
-        trust.set_args(self.args)
-        trust.discover_sigstore(fq_name)
-        write_out("Pulling {} ...".format(fq_name))
-        skopeo_copy("docker://{}".format(fq_name), image,
-                    debug=self.args.debug, insecure=insecure,
-                    policy_filename=self.policy_filename)
+        if self.args.image.startswith("dockertar:"):
+            path = self.args.image.replace("dockertar:", "", 1)
+            check_call(["docker", "load", "-i", path])
```

Comment:
Thanks for the suggestion, updated below ⬇️. I didn't change to using `return 0`, since it doesn't add much here. I see this more as the docker backend equivalent of the if-else cascade in `syscontainers.py`. E.g. we might support more prefixes in the future.